packages/web/src/styles/interaction.css

```
@@ -180,176 +124,98 @@
+    padding: 0.6rem 0.75rem;
+    background: var(--public-paper-raised);
+    color: #b6604f;
 }
 
 .spinner {
     width: 1rem;
     height: 1rem;
-    border: 2px solid
-        color-mix(in srgb, var(--fn-text-primary) 20%, transparent 80%);
-    border-top-color: var(--fn-text-primary);
+    border: 2px solid color-mix(in srgb, currentColor 25%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)